__tests__/app.test.js

```diff
@@ -97,3 +97,66 @@ describe("4. GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("5. PATCH /api/articles/:article_id", () => {
+  test("status 200: responds with an updated article object incrementing the votes", () => {
+    const article_id = 1;
+    const updatedArticle = { inc_votes: 1 };
+    return request(app)
+      .patch(`/api/articles/${article_id}`)
+      .send(updatedArticle)
+      .expect(202)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        expect(body).toEqual(
+          expect.objectContaining({
+            article_id: 1,
+            title: "Running a Node App",
+            topic: "coding",
+            author: "jessjelly",
+            body: "This is part two of a series on how to get up and running with Systemd and Node.js. This part dives deeper into how to successfully run your app with systemd long-term, and how to set it up in a production environment.",
+            created_at: "2020-11-07T06:03:00.000Z",
+            votes: 1,
+          })
+        );
+      });
+  });
+  test("status 200: responds with an updated article object decrementing the votes", () => {
+    const article_id = 4;
+    const updatedArticle = { inc_votes: -10 };
+    return request(app)
+      .patch(`/api/articles/${article_id}`)
+      .send(updatedArticle)
+      .expect(202)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        expect(body).toEqual(
+          expect.objectContaining({
+            article_id: 4,
+            title: "Making sense of Redux",
+            topic: "coding",
+            author: "jessjelly",
+            body: "When I first started learning React, I remember reading lots of articles about the different technologies associated with it. In particular, this one article stood out. It mentions how confusing the ecosystem is, and how developers often feel they have to know ALL of the ecosystem before using React. And as someone who’s used React daily for the past 8 months or so, I can definitely say that I’m still barely scratching the surface in terms of understanding how the entire ecosystem works! But my time spent using React has given me some insight into when and why it might be appropriate to use another technology — Redux (a variant of the Flux architecture).",
+            created_at: "2020-09-11T20:12:00.000Z",
+            votes: -10,
+          })
+        );
+      });
+  });
+  test("status 404: responds for invalid paths in article ids", () => {
+    return request(app)
+      .get("/api/articles/50000000")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("No article found for article_id: 50000000");
+      });
+  });
+  test("status 400: responds for bad requests", () => {
+    return request(app)
+      .get("/api/articles/hi")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad request: Invalid input");
+      });
```

Comment:
Would also want tests for:
- if the POST body had the wrong value aka a string instead of an integer on the key of `inc_votes`
- if the POST body was missing the `inc_votes` key